pkg/stargate/deployments.go

```diff
@@ -138,6 +138,9 @@ func NewDeployments(stargate *api.Stargate, dc *cassdcapi.CassandraDatacenter) m
 				},
 			},
 		}
+		if klusterName, found := stargate.Labels[api.K8ssandraClusterLabel]; found {
```

Comment:
I like the variable name 😄

Reply:
Speaking of which, have you considered adding a short name to the `k8ssandracluster` CRD? Maybe `kluster`?